openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -276,3 +276,39 @@ def get_queryset(self):
             )
 
         return invitations
+
+
+class UpdatePermissions(ListView):
+    """
+    UpdatePermissions View
+    """
+    def get(self, request, *args, **kwargs):
+        """
+        It will get invitation object of the user and give permissions which
+        are assigned to that user in invitation object
+
+        Returns:
+            It will redirect to admin dashboard if user is active and have permissions
+            otherwise it will redirect to home page
+        """
+        user = self.request.user
+        invitation_obj = Invitation.objects.filter(email=user.email).first()
```

Comment:
Let's create a manager class for Invitation model and write a method in it to return the invitation based on email as this statement is also being used in `get_registration_form_override`

Reply:
If it had been written at a single place, it would have been fine but it is being duplicated, that is why I suggested this, plus we already have such manager methods which comprise of a single filter or select related etc.

However, it is a suggestion, let's take input form other reviewers.